tenant-file/portal-app/src/component/add-property.tsx

```diff
@@ -0,0 +1,178 @@
+                            <option value="IA">Iowa</option>
+                            <option value="KS">Kansas</option>
+                            <option value="KY">Kentucky</option>
+                            <option value="LA">Louisiana</option>
+                            <option value="ME">Maine</option>
+                            <option value="MD">Maryland</option>
+                            <option value="MA">Massachusetts</option>
+                            <option value="MI">Michigan</option>
+                            <option value="MN">Minnesota</option>
+                            <option value="MS">Mississippi</option>
+                            <option value="MO">Missouri</option>
+                            <option value="MT">Montana</option>
+                            <option value="NE">Nebraska</option>
+                            <option value="NV">Nevada</option>
+                            <option value="NH">New Hampshire</option>
+                            <option value="NJ">New Jersey</option>
+                            <option value="NM">New Mexico</option>
+                            <option value="NY">New York</option>
+                            <option value="NC">North Carolina</option>
+                            <option value="ND">North Dakota</option>
+                            <option value="OH">Ohio</option>
+                            <option value="OK">Oklahoma</option>
+                            <option value="OR">Oregon</option>
+                            <option value="PA">Pennsylvania</option>
+                            <option value="RI">Rhode Island</option>
+                            <option value="SC">South Carolina</option>
+                            <option value="SD">South Dakota</option>
+                            <option value="TN">Tennessee</option>
+                            <option value="TX">Texas</option>
+                            <option value="UT">Utah</option>
+                            <option value="VT">Vermont</option>
+                            <option value="VA">Virginia</option>
+                            <option value="WA">Washington</option>
+                            <option value="WV">West Virginia</option>
+                            <option value="WI">Wisconsin</option>
+                            <option value="WY">Wyoming</option>
+                        </Field>
+                        <label style={{textIndent: '99px'}} htmlFor="zip">*Zip Code:</label>
+                        <Field id="zip" name="zip" /* component={StyledInput} */ /* validate={ValidateRequiredField} */ />
+
+                        <button>Click Here to Submit</button>
+                    </FormColumn>
+                </FormContainer>
+
+                
+            </Form>
+        )}
+    </Formik></AddPropertyWrapper>
+}
+ReactDOM.render(<form />, document.getElementById('root'));
```

Comment:
What is this for?

Reply:
Hey Ryan - I could be wrong, but I think the "ReactDOM.render" line is needed in order for the form to display.  I just copied it from the documentation: https://formik.org/docs/examples/basic
If you want me to check whether it works without it though I can try.  Thanks.

Reply:
I think is for a standalone react solution, you can see that we have that call in our index.tsx. It's required to start up a react app. In our case you can safely ditch this line because your default export enables us to render it.